documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -1,45 +1,193 @@
+
+### End of Term Summary
+
+At the end of each term, the council is encouraged to submit a "summary"  which describes the funding decisions the councils have made and their reasoning for these decisions. This summary will act as an assessment of the council and will be the information point for governance to decide whether to re-elect the council. This summary should be made as transparant as pos
+
+## Mechanism
+
+The governance proposal PGFCouncil elects the multisignature account address of the PGF council.
+
+The PGF council members will (by definition) be in charge of a "k of n" multisignature account whereby the $\text{10% inflation} * \text{spending_cap}$ is allocated and spent from the `PGF` internal address.
+
+Members of the PGF council will then unilaterally be able to propose and sign transactions for this purpose. Consensus on these transactions, in addition to motivation behind them will be handled off-chain, and should be recorded for the purposes of the "End of Term Summary".
+
+
+### cPGF (continuous PGF)
+
+<!-- These transactions will be more involved than the rPGF transactions (which will act as simple transactions).
+
+
+```rust
+struct cPGFTx {
+    // Source address - the multisig
+    address: Address,
+    // Asset identifier for this input - will always be NAM
+    token: NAM,
+    // Asset value in the input, will be distributed each epoch
+    amount_epoch: u64,
+    // A signature over the hash of the transaction
+    sig: Signature,
+    // Used to verify the owner's signature
+    pk: PublicKey,
+}
+```
+
+```rust!
+struct TxOut {
+    // Destination addresses (can be a vector)
+    address: Vec<Address>,
+    // Asset identifier for this output
+    token: NAM,
+    // Asset value in the output, will happen each epoch
+    amount: u64,
+}
+``` -->
+
+The following data is attached to the PGF transaction and will allow the counsil to decide which projects will be continously funded. Each tuple reppresent the address and the respecting amount of NAM that will receive every epoch. The list of project will be stored in storage under the PGF internal address substorage space.
+
+```rust
+struct PFGTxData {
+    projects: HashSet<(Address, u64)>
+}
+```
+
+### rPGF (retroactive PGF)
+No different logic compared to normal `Transfer` txs, except that it should allow for multiple receivers (see above).
+
+
+## Addresses
+Governance adds 1 internal address:
+
+`PGF` internal address
+
+The internal address VP will be allocated the 10% inflation of NAM. This will be added in addition to what was unspent by the previous council.
+It will also implement the logic required in order to allow the cPGF transactions to be made. I.e finalize-block, which will send the addresses their respective amounts each end-of-epoch.
+
+The Governance address should also allow the burning of funds by the council.
```

Comment:
I don't think this requires special handling

Reply:
or, is there an argument for special handling?

Reply:
Regarding burning